tests/modules/hamronization/amrfinderplus/test.yml

```diff
@@ -0,0 +1,8 @@
+- name: hamronization amrfinderplus test_hamronization_amrfinderplus
+  command: nextflow run ./tests/modules/hamronization/amrfinderplus -entry test_hamronization_amrfinderplus -c ./tests/config/nextflow.config  -c ./tests/modules/hamronization/amrfinderplus/nextflow.config
+  tags:
+    - hamronization
+    - hamronization/amrfinderplus
+  files:
```

Comment:
You want to add the md5sum of `versions.yml` and JSON output?

Reply:
No, if it is not needed ...